### PR TITLE
Enable tests for app-shells/autojump

### PR DESCRIPTION
**Pull request by mattst88**
To do this, I needed to change `dev-vcs/pre-commit` from using `python-single-r1` to `python-r1`.

Closes: https://bugs.gentoo.org/530742

### CODE REVIEW

File: profiles/arch/x86/package.use.mask [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Matt Turner <mattst88@gentoo.org> (2021-05-31)
```
Comment (mgorny) [inline profiles/arch/x86/package.use.mask:4]: Just add `~x86`, nobody cares anyway.

**Review by mgorny — COMMENTED** [review]
Well, I suppose you could make autojump single-impl but I suppose either way is fine.

Now, I wouldn't do all that profile masking because that's going to be PITA in the future. Just revbump it and request rekeywording.

File: profiles/arch/x86/package.use.mask (hunk 1 above)
Comment (mattst88) [reply profiles/arch/x86/package.use.mask:4]: Fair point wrt keywording and revbump vs masking. Will do

**Comment by mattst88** [comment]
> Well, I suppose you could make autojump single-impl but I suppose either way is fine.

I actually wanted to do this, but I don't know how to handle this bit with python-single-r1:

````
	dobin bin/"${PN}"
	python_replicate_script "${ED}"/usr/bin/"${PN}"
````

since autojump doesn't have a standard build system. Any suggestions?

**Comment by mgorny** [comment]
```
python_doscript ...
```

(in python-r1 you'd add `python_foreach_impl ...`)

**Pull request closed without merging**